src/Stratis.Bitcoin/Configuration/VersionProvider.cs

```diff
@@ -7,8 +8,16 @@ public class VersionProvider : IVersionProvider
     {
         public string GetVersion()
         {
-            Match match = Regex.Match(this.GetType().AssemblyQualifiedName, "Version=([0-9]+\\.[0-9]+\\.[0-9]+\\.[0-9])");
-            return match.Groups[1].Value;
+            Match match = Regex.Match(this.GetType().AssemblyQualifiedName, "Version=([0-9]+)(\\.([0-9]+)|)(\\.([0-9]+)|)(\\.([0-9]+)|)");
```

Comment:
Let's say that a user sees 2 nodes with versions: 3.0.2.2 and 3.0.2.
Will it be really obvious that 3.0.2 means 3.0.2.0?
For me I'd interpret this as 3.0.2.2 is a patched version and 3.0.2 is the release version.

Reply:
Commit is already in but - It's not obvious that 3.0.2 is the same as 3.0.2.0 to a non-technical users. Clarity and consistency is key especially when you are in a regulated industry like healthcare and financials.